### ISSUE TITLE
Cuckoo Package: MISP processing

### ISSUE THREAD

**Issue by SparkyNZL** [cuckoosandbox/cuckoo#1458]
The MISP processing is working really well, only issue is, is that its pulling stuff out of "Analysis Comments" which is causing a lot of IOCs to be listed (eg 17000 have 8.8.8.8  in a comment field ) this causes addition of these to the report which is not that helpful :( 

Perhaps a future improvement 

:)

**Comment by doomedraven** [comment]
Check whitelist it should support it to ban them

**Comment by SparkyNZL** [comment]
yeah thats what i thought too, but either its not working or im being a retard, but most likely the latter

ive put in  "/8.8.8.8/"

**Comment by doomedraven** [comment]
Why do you put slashes? Just ip without " also, and if that not work put some print in loading whitelist in misp module to verify if that loads correctly

**Comment by SparkyNZL** [comment]
in MISP you have to put it in as valid regex so it needs it ?

**Comment by doomedraven** [comment]
oh i see, maybe @jbremer removed it %) no idea how it disappeared 

```
whitelist = list()
# load whitelist if exists
if os.path.exists(os.path.join(CUCKOO_ROOT, "conf", "misp.conf")):
      whitelist = Config("misp").whitelist.whitelist
      if whitelist:
          whitelist = [ioc.strip() for ioc in whitelist.split(",")]
```

the config format should be updated to new format as `config("mist:whitelist:whitelist")`

**Comment by SparkyNZL** [comment]
Oh I was whitelisting in misp, you need to use regex php for that hence the
format.

The domain whitelist is only that, it doesn't stop the matching in misp of
https domains ir ip-addresses resolved from the whitelisted domains,

Which goes back to a ticket where @jbremer and I were discussing this,
because all the stuff excluded in the whitelist ends up in moloch as well
as othing network areas in cuckoo like https, http, dns  this needs to be
looked at later down the track as windows is generating more and more noise
 ;(

**Comment by doomedraven** [comment]
i will send you noise shutdown in slack

**Comment by denmilu** [comment]
@doomedraven can you add me to slack group? my slack nick: tranpnam
thank you very much!

**Comment by doomedraven** [comment]
no, i don't have right for that, there is public IRC ;)

**Comment by jbremer** [comment]
@denmilu If you want I can add you to our Slack, though!

**Comment by denmilu** [comment]
@jbremer Very happy to hear that. My Slack is: tranpnam . Thank you very much!
